src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -433,6 +433,19 @@ mongoc_client_encryption_datakey_opts_set_keymaterial (
  * Explicit Encryption options.
  *--------------------------------------------------------------------------
  */
+struct _mongoc_client_encryption_encrypt_range_opts_t {
+   struct {
+      bson_value_t value;
+      bool set;
+   } min;
+   bson_value_t max;
```

Comment:
```suggestion
   struct {
      bson_value_t min;
      bson_value_t max;
      bool set;
   } minmax;
```

To clarify `set` applies to both `min` and `max`.